hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/HxInsightClient.java

```diff
@@ -26,14 +26,16 @@
 
 package org.alfresco.hxi_connector.hxi_extension.service;
 
+import static java.lang.String.format;
+
 import static org.apache.http.HttpStatus.SC_ACCEPTED;
 import static org.apache.http.HttpStatus.SC_OK;
 import static org.apache.http.HttpStatus.SC_SERVICE_UNAVAILABLE;
 
 import static org.alfresco.hxi_connector.hxi_extension.service.model.FeedbackType.RETRY;
 import static org.alfresco.hxi_connector.hxi_extension.service.util.HttpUtils.ensureCorrectHttpStatusReturned;
 
-import java.io.IOException;
+import java.io.*;
```

Comment:
It'd be neater to import exactly what we need rather than importing the whole package.

Reply:
I'm surprised PMD scan didn't catch that. I thought we had a rule for that.